src/explorer/CosmosDBTreeItem.ts

```diff
@@ -215,6 +265,32 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         return new Map([[appSettingKey, database.connectionString]]);
     }
 
+    private async promptForPostgresAppSettings(appSettingsDict: StringDictionary, database: DatabaseTreeItem): Promise<Map<string, string>> {
```

Comment:
Kind of the same idea here.  It's pretty much the same code as `promptForDocDBAppSettings`

Reply:
I was able to reduce the promptForAppSettings functions, it is available in the latest commit.